pkgs/servers/nextcloud/packages/generate.sh

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -I nixpkgs=../../../.. -i bash -p nc4nix
+#!nix-shell nc4nix-shell.nix -i bash
 
 set -e
 set -u
 set -o pipefail
 set -x
```

Comment:
```suggestion
set -eou pipefail
```
running scripts always with -x is usually very noisy